### DISCUSSION TITLE
Using multiple notes as Considered Text

### DISCUSSION THREAD

**Discussion by Innovator-X** [nhaouari/obsidian-textgenerator-plugin#27] [Ideas]
I have a few use cases where I need to generate a summary of text that is located in several separate notes. For example, I have a summary of activities in each Daily Note, and would like to generate a summary for the week without manually copying the text from 7 different notes. I'm interested to hear if you have already considered this, if anyone else has similar use cases, or if it's even possible to add that functionality.

Thanks for your great work on this plugin!

**Comment by nhaouari** [comment]
This is possible using Templates in new version of the plugin 0.1.x, you need to have a page with the links of the notes you want to include and a template that consider the "children" of the note. see [https://text-gen.com/context-available-for-templates](https://text-gen.com/context-available-for-templates ).

**Comment by VictorJermiin** [comment]
Is there any way that you could do something like this with Templates?
<img width="477" alt="Pasted image 20230114190403" src="https://user-images.githubusercontent.com/45830483/212490707-f6c8a7de-4bee-47a6-bb41-39847eaa0634.png">
